util/cookies.ts

```diff
@@ -1,37 +1,14 @@
 import cookie from 'cookie';
-import Cookies from 'js-cookie';
-
-// more robust way to get items from cookies without parsing all the time
-export function getParsedCookie(key: string): CookieValue | undefined {
-  const cookieValue = Cookies.get(key);
-
-  if (!cookieValue) {
-    return undefined;
-  }
-
-  try {
-    return JSON.parse(cookieValue); // Type should be a string
-  } catch (err) {
-    return undefined;
-  }
-}
+import { cookies } from 'next/headers';
 
 export type CookieValue = {
   id: number;
-  stars: number;
+  comment: string;
 }[];
 
-// more robust way to set items to set the cookie without stringify all the time
-export function setStringifiedCookie(key: string, value: CookieValue) {
-  Cookies.set(key, JSON.stringify(value));
-}
-
-export function deleteCookie(key: string) {
-  Cookies.remove(key);
-}
-
-export function stringifyCookieValue(value: CookieValue) {
-  return JSON.stringify(value);
+// more robust way to get items from cookies without parsing all the time
+export function getCookie(name: string): string {
+  return cookies().get(name)?.value || '[]';
```

Comment:
this seems like an unusual default value - probably should be `undefined` (you can just get rid of the `|| '[]'`)

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/ccaf7ec58f04066b612c7338707027caec1386ae/util/cookies.ts#L11